### PR TITLE
Added a MIT licence file

### PR DESCRIPTION
**Pull request by rogdevil**

### CODE REVIEW

**Comment by gokatz** [comment]
Thank you so much for pitching in @sep2001aryan.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Amol Grover
```
Comment (gokatz) [inline LICENSE:3]: Hi @sep2001aryan. Thanks again for your PR. One little doubt for me here is the name **Amol Grover**

Comment (gokatz) [reply LICENSE:3]: Any reason for this name?

**Comment by gokatz** [comment]
Your contribution is really valuable. Unfortunately, the changes have been done in a series of other PRs. If you are still interested, kindly pick up an another issue.

File: LICENSE (hunk 1 above)
Comment (rogdevil) [reply LICENSE:3]: I forgot to change the name on license. I am getting started 😅 sorry for that

Comment (sep2001aryan) [reply LICENSE]: I can change it would accept the PR ?

Comment (gokatz) [reply LICENSE:3]: No issues, @sep2001aryan. Glad you made a PR in this repo. If you need any help in contributing, please reach me out.

**Pull request closed without merging**